pymagicc/__init__.py

```diff
@@ -10,28 +10,26 @@
 
 # -*- coding: utf-8 -*-
 
-from .compat import get_param
-from .paths import _get_magicc_paths
-from .api import MAGICC
-
 import datetime
 import linecache
 import logging
 import os
-import platform
 import subprocess
 
 import f90nml
 import pandas as pd
 
 from ._version import get_versions
+from .config import config as _config
+from .api import MAGICC6, MAGICC7  # noqa
+
 __version__ = get_versions()["version"]
 del get_versions
 
-_magiccpath, _magiccbinary = _get_magicc_paths()
-
-_WINDOWS = platform.system() == "Windows"
+# default parameters and cannot be changed after module load
```

Comment:
Can you elaborate the use case here?

Why not write a script like

```python
_magiccpath = "/tmp/magicc-super-fast"
out = run(testscen)

_magiccpath = "/tmp/magicc-test"
test_out = run(testscen)
```

Reply:
The less changing of module-level variables the better. Module level variables like _magiccpath don't make good configuration sources when other variables depend on them. They work great as constants, but you can end up with all kinds of dangling references to previous values. For example, changing _magiccpath doesn't change `rcp26` and the other module-level imports since they are eagerly loaded when the module is first imported.
 
In this case the configuration about the target distribution is determined when an instance of MAGICC is instantiated. Then, any variables when depending on the configuration are loaded.

My preference would be to remove _magiccpath and _magiccbinary as they no longer make sense when dealing with multiple distributions.

Reply:
> They work great as constants, but you can end up with all kinds of dangling references to previous values. 
Very good points ... @znicholls and I discussed a bit reading IN files yesterday, and when loading these lazily (since there are many in MAGICC7), it would be good to be able to have them follow path settings as well.

If you see a way of removing _magiccpath and _magiccbinary I'd be happy with that -- also better to have fewer places to change config!